rq/registry.py

```diff
@@ -234,7 +251,7 @@ def cleanup(self, timestamp: Optional[float] = None):
 
                     else:
                         job.set_status(JobStatus.FAILED)
-                        job._exc_info = "Moved to FailedJobRegistry at %s" % datetime.now()
+                        job._exc_info = "Moved to FailedJobRegistry, due to job expiry, at %s" % datetime.now()
```

Comment:
Can we update this message to specifically mention `AbandonedJobError`? Also please see https://github.com/rq/rq/pull/1840 to see how the docs can be improved.

Reply:
Done, regarding docs, @luto WDYT?

Reply:
Looks good to me! All messages and the docs now call those jobs "abandoned", which is fitting imho. The most important thing for me is that you can find the docs based on the message. Both talk about `AbandonedJobError`, so that works. Thanks! :)

Reply:
Thank you 🤝